app/route_helpers.py

```diff
@@ -0,0 +1,19 @@
+
+from flask import make_response, abort
+
+
+
+def validate_model_id(cls, id):
```

Comment:
Nice helper function.  It would be good to have another to validate required fields in the request body.